client/src/components/core/ms-toggle/MsGridListToggle.vue

```diff
@@ -11,9 +11,6 @@
       @click="$emit('update:modelValue', modelValue === DisplayState.Grid ? DisplayState.List : DisplayState.Grid)"
     >
       <ion-icon :icon="grid" />
-      <span v-if="modelValue === DisplayState.Grid">
-        {{ $t('WorkspacesPage.viewDisplay.grid') }}
```

Comment:
Is this text used somewhere else? If not, it should be removed from the translation files.

Reply:
Yes sure! We'll probably need them in the future, but for the moment we can do away with them.